e2e/accounts/account_node_test.go

```diff
@@ -0,0 +1,647 @@
+package accounts
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/status-im/status-go/e2e"
+	"github.com/status-im/status-go/geth/account"
+	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/node"
+	"github.com/status-im/status-go/geth/params"
+	. "github.com/status-im/status-go/testing"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	tmpDirName        = "status-tmp"
+	account1          = "test-account1.pk"
+	account2          = "test-account2.pk"
+	accountEIP55      = "test-account1-before-eip55.pk"
```

Comment:
Name `accountEIP55` is actually an opposite to `account before EIP55` meaning. Confused a bit when I met it in code down here. Maybe rename to `accountPreEIP55` or `accountBeforeEIP55` (longer, but clearer, or maybe there is a better name)

Reply:
Ok, changed